### PR TITLE
Upgraded to assembly-image 1.7.1.

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ gem 'turbolinks' # improves speed of following links in web application
 gem 'uglifier' # compressor for JavaScript assets
 
 # Stanford gems
-gem 'assembly-image'
+gem 'assembly-image', '~> 1.7.1'
```
Comment (jcoyne) [inline Gemfile]: Why are you pinning so pessimistically?  Would `~> 1.7` suffice?

Comment (justinlittman) [reply Gemfile]: Fixed. More optimistic.

**Comment by jcoyne** [comment]
@justinlittman looks like you need to `bundle install` and check in the resulting `Gemfile.lock`

**Comment by justinlittman** [comment]
Some day I'll get the hang of this.

**Pull request merged**